R/model.R

```diff
@@ -64,29 +64,43 @@ winter_run_model <- function(scenario = NULL, seeds = NULL){
     egg_to_fry_surv <- surv_egg_to_fry(
       proportion_natural = 1 - proportion_hatchery,
       scour = prob_nest_scoured,
-      temperature_effect = mean_egg_temp_effect
+      temperature_effect = rep(0.6466230, 31)
     )
```

Comment:
make sure to change this hard coded value so that it exists in the appropriate data and use that insetad